src/openms/source/APPLICATIONS/ToolHandler.cpp

```diff
@@ -107,8 +107,6 @@ namespace OpenMS
     tools_map["NoiseFilterGaussian"] = Internal::ToolDescription("NoiseFilterGaussian", "Signal processing and preprocessing");
     tools_map["NoiseFilterSGolay"] = Internal::ToolDescription("NoiseFilterSGolay", "Signal processing and preprocessing");
     tools_map["OMSSAAdapter"] = Internal::ToolDescription("OMSSAAdapter", "Identification");
-    tools_map["OpenPepXL"] = Internal::ToolDescription("OpenPepXL", "Identification");
```

Comment:
hmm I thought we had in our changelog that these are now tools..

Reply:
Ok, I did not know that. I used the minimal number of changes to make it consistent:
They had the 'bool official = false' set in TOPPBase and were still named '@page UTILS_..." in Doxygen, but were in the wrong category in ToolHandler. All of these need to be consistent for our docs to work properly.

Reply:
@enetz can you fix this?

Reply:
@timosachsenberg @cbielow It is very possible that I forgot some details when turning my tools from UTILs to TOOLs. Probably also applies to OpenPepXLLF and XFDR, not just OpenPepXL. I will look into it tomorrow.